workday/13.0/modules/ROOT/pages/workday-connector-examples.adoc

```diff
@@ -0,0 +1,349 @@
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration.
+
+If you don't know how to use a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure *HTTP Listener*:
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for *http* and select the *Listener* operation:
++
+.Select the Listener operation in the Mule Palette view.
+image:workday-select-listener.png[Select the Listener operation]
++
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Click *OK* to accept the defaults.
+. Set the *Path* field to `/`:
++
+.Set the Path field in the Listener configuration.
+image::workday-http-props.png[Listener configuration]
+
+=== Add a Transform Message Component to Attach the Metadata
+
+To add and configure a Transform Message component:
+
+. In the *Mule Palette* view, search for `transform` and select *Transform Message*:
++
+.Select Transform Message in the Mule Palette view.
+image::workday-select-transform.png[Select the Transform Message component ]
++
+. Drag the *Transform Message* component onto the canvas, to the right of the *Listener* component.
+. In the *Transform Message* configuration, overlay the brackets in the *Output* section with this XML:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/xml
+ns ns0 urn:com.workday/bsvc
+---
+{
+  ns0#Put_Fund_Request @(ns0#Add_Only: true, ns0#version: "v35.0"): {
+    ns0#Fund_Data: {
+      ns0#Fund_Name: "I.M.F",
+      ns0#Fund_Type_Reference: {
+        ns0#ID @(ns0#'type': 'Fund_Type_ID'): "FUND_TYPE-6-3"
+      }
+    }
+  }
+}
+----
+
+The following XML shows sample input to the *Transform Message* component
+
+[source,xml,linenums]
+----
+<ns0:Put_Fund_Request xmlns:ns0="urn:com.workday/bsvc" Add_Only="true" version="v35.0">
+  <ns0:Fund_Data>
+    <ns0:Fund_Name>I.M.F</ns0:Fund_Name>
+    <ns0:Fund_Type_Reference>
+      <ns0:ID type="Fund_Type_ID">FUND_TYPE-6-3</ns0:ID>
+    </ns0:Fund_Type_Reference>
+  </ns0:Fund_Data>
+</ns0:Put_Fund_Request>
+----
+
+The following screenshot shows the corresponding output, as it appears in the *Output* section of the *Transform Message* configuration:
+
+.Output in the Transform Message configuration
+image::workday-transform-message.png[Transform Message output]
+
+== Add and Configure the Financial Management Operation
+
+. In the *Mule Palette* view, search for *workday* and select the *Financial management* operation.
+. Drag the *Financial management* operation onto the canvas, to the right of the *Transform Message* component.
```

Comment:
Is it Financial management or Financial Management? Two diff ways on lines 100 and 101